gui/gui/src/TGSpeedo.cxx

```diff
@@ -71,6 +72,9 @@ TGSpeedo::TGSpeedo(const TGWindow *p, int id)
    fImage = TImage::Open(fPicName);
    if (!fImage || !fImage->IsValid())
       Error("TGSpeedo::Build", "%s not found", fPicName.Data());
+   fBufferCount = 0;
+   fBufferSize = 10;
```

Comment:
fBufferSize = 0; // for backward compatibility

Reply:
OK